Testscripts/Windows/VERIFY-XDP-ACTION.ps1

```diff
@@ -10,8 +10,6 @@
 param([object] $AllVmData,
     [object] $CurrentTestData)
 
-$MIN_KERNEL_VERSION = "5.6"
-$RHEL_MIN_KERNEL_VERSION = "4.18.0-213"
 $iFaceName = "eth1"

```

Comment:
Is it always eth1

Reply:
With the testcase config we have added an entra nic. With my observation and which is backed by @haiyangz about the name of Synthetic nic would always follow eth`n` format. VF name that is not stable but we can rely on Synthetic NIC